src/Components/Task.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+
+const Task = ({ description, setTaskList, tasks, id, setTaskToBeEdited, setNewDescription }) => {
+    const [isStrikeThrough, setStrikeThrough] = useState(false);
+
+    const removeTask = (id) => {
+        return setTaskList(tasks.filter(task => task.id !== id))
+    };
+
+    useEffect(() => {
+        const modal = document.querySelector('.modal');
+        const overlay = document.querySelector('.overlay');
+        const editButtons = Array.from(document.querySelectorAll('.button__edit'));
```

Comment:
We try to avoid this type of code as it is not a good practice. Instead we can handle this in the `return` block in JSX if we want to create edit buttons.